scripts/workers/code_upload_submission_worker.py

```diff
@@ -684,6 +694,7 @@ def main():
             ) and not message_body.get(
                 "is_static_dataset_code_upload_submission"
             ):
+                time.sleep(35)
```

Comment:
Why this big wait time?

Reply:
@Ram81 This is to guarantee that submission worker reads this message and evaluates it. Waittime of a message is 30 secs in SQS by default. This approach removes the probablistic approach is more inclined towards deterministic and which increase speed.
Tested this approach and the whole submission lifecycle time was reduced drastically due to this.